docs/TRANSLATORS.md

```diff
@@ -26,7 +26,7 @@ To create a translation to a language not yet available, start by creating the
 initial template:
 
 ```
-$ meson compile -C build/ systemd-pot
+$ ninja -C build/ systemd-pot
```

Comment:
these were changed intentionally in https://github.com/systemd/systemd/pull/17939 - any specific reason to switch back? If not, please drop this commit

Reply:
For me, it doesn't run the old version. I'll remove this commit, but a check is needed.